plugins/query-insights/src/main/java/org/opensearch/plugin/insights/core/exporter/DebugExporter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.insights.core.exporter;
+
+import org.opensearch.plugin.insights.rules.model.SearchQueryRecord;
+
+import java.util.List;
+
+/**
+ * Debug exporter for development purpose
+ */
+public final class DebugExporter extends AbstractExporter {
+    /**
+     * Constructor of DebugExporter
+     */
+    public DebugExporter() {}
```

Comment:
Default exporter would be null, the debug exporter is more about development and testing purpose.

Reply:
I meant the default constructor is implicit. Do we need to explicitly specify it?

Reply:
Yes, as I memtioned in the previous comment, since we enabled JavaDoc check for the whole query insights plugin, this is needed to make the checker happy. Otherwise we get
```
/Users/cyji/OpenSearch/myfork/OpenSearch/plugins/query-insights/src/main/java/org/opensearch/plugin/insights/core/exporter/DebugExporter.java:20: error: DebugExporter.<init> (constructor): javadocs are missing
public final class DebugExporter implements AbstractExporter {
             ^
```

Reply:
Weird. I've never encountered this issue myself. Is there no way to tune the javadoc check to get the behavior you want?

Reply:
It's a "feature": https://bugs.openjdk.org/browse/JDK-8249634